components/nav.js

```diff
@@ -59,7 +63,7 @@ const Nav = () => {
             {buttons.map(({ key, href, label }) => (
               <div key={key} className='db center w-auto fr ph2'>
                 <Link href={href} key={key}>
-                  <a className='btn-alt no-underline black pv3 ph4 br3'>{label}</a>
+                  <a className='btn-alt no-underline black pv3 ph4 br3' style={{fontSize:25 }}>{label}</a>
```

Comment:
i think this is far too big (not sure if the design pixel density is off) let's revert this file